charts/mirrorbits/tests/custom_values_test.yaml

```
@@ -32,3 +37,31 @@ tests:
       - equal:
           path: spec.selector["app.kubernetes.io/instance"]
           value: "RELEASE-NAME-files"
+  - it: Should define a customized "files" deployment
+    template: deployment.files.yaml
+    set:
+      image:
+        files:
+          pullPolicy: Never
+    asserts:
+      - hasDocuments:
+          count: 1
+      - isKind:
+          of: Deployment
+      - equal:
+          path: "spec.template.spec.containers[*].imagePullPolicy"
+          value: Never
+  - it: Should define a customized "mirrorbits" deployment
```

```suggestion
  - it: should define a customized "files" deployment
    template: deployment.files.yaml
    set:
      image:
        files:
          pullPolicy: Never
    asserts:
      - hasDocuments:
          count: 1
      - isKind:
          of: Deployment
      - equal:
          path: "spec.template.spec.containers[*].imagePullPolicy"
          value: Never
  - it: should define a customized "mirrorbits" deployment
```
Nit, same as the existing ones and as the doc examples from helm-unittest.